README.md

```diff
@@ -229,11 +229,14 @@ The following properties are expected to be present on the deployment `context`
 
 ## Configuring Amazon S3
 
-### Minimum S3 Permissions
+### Deployment user and S3 permissions
 
-Ensure you have the minimum required permissions configured for the user (accessKeyId). A bare minimum policy should have the following permissions:
+The environment in which the `ember deploy` command is run (outside of development) needs to have an AWS account with a policy that allows writing to the S3 bucket.
+The best way to do this is to create an IAM user to be the "deployer", and place its security credentials (Access Key ID and Access Secret) in the environment on the server or CI environment where deployment takes place.
```

Comment:
maybe replace `server` with `machine` to address the concern I was expressing above? (open to suggestions on wording)